config/locales/en.yml

```diff
@@ -1292,6 +1292,8 @@ en:
       your_appeal_rejected: Your appeal has been rejected
   edit_profile:
     basic_information: Basic information
+    errors:
+      fields_with_values_missing_names: Names of extra profile fields with values cannot be empty
```

Comment:
This is a common footgun, Rails will combine the message string here with the localized attribute name (`:fields`) so the printed out message will be "Fields Names of extra profile fields with values cannot be empty"

Reply:
Good to know, thanks! I modified the message to be a sentence beginning with "Fields".